src/extensions/renderer-api/kube-object-status.ts

```diff
@@ -0,0 +1 @@
+export { KubeObjectStatus as Status, KubeObjectStatusLevel as Level } from "../registries/kube-object-status-registry"
```

Comment:
Does not look right? We should not use generic `Status`/`Level` if these are exposed within `K8sApi` namespace.

Reply:
Yes, this is still wrong here. Will fix

Reply:
Done